test/Microsoft.AspNet.Mvc.Core.Test/ActionResults/ObjectResultTests.cs

```diff
@@ -32,25 +33,25 @@ public static IEnumerable<object[]> ContentTypes
                         };
 
                 // Empty accept header, should select based on contentTypes.
-                yield return new object[] { contentTypes, "", "application/json;charset=utf-8" };
+                yield return new object[] { contentTypes, "", "application/json; charset=utf-8" };
```

Comment:
wondering why this line needed to change? do we expect a space between the parameters?